internal/core/analysis_api/handlers/list.go

```diff
@@ -97,9 +103,12 @@ func pythonListFilters(enabled *bool, nameContains, severity string, types, tags
 		filters = append(filters, typeFilter)
 	}
 
-	if severity != "" {
-		filters = append(filters, expression.Equal(
-			expression.Name("severity"), expression.Value(severity)))
+	if len(severity) > 0 {
+		severityFilter := expression.AttributeNotExists(expression.Name("severity"))
+		for _, severityType := range severity {
+			severityFilter = severityFilter.Or(expression.Contains(expression.Name("severity"), string(severityType)))
+		}
+		filters = append(filters, severityFilter)
```

Comment:
If we don't want to do the first expression hack, you can do it like this:

```suggestion
	if len(severity) > 0 {
		severityFilter := expression.Equal(expression.Name("severity"), string(severity[0]))
		for _, severityType := range severity[1:] {
			severityFilter = severityFilter.Or(expression.Equal(expression.Name("severity"), string(severityType)))
		}
		filters = append(filters, severityFilter)
	}
}
```

And then go ahead and make this change for the other loop checks as well (e.g. resource types)